README.md

```diff
@@ -235,6 +235,6 @@ I would like to thank the following people for their help, ideas and contributio
 * ![legendaryhobo1](https://dcbadge.vercel.app/api/shield/510589560502091787?style=flat-square&?compact=true&?logoColor=orange) (Ideas + Alpha and Beta testing for me :sunglasses:)
 * ![natey1.](https://dcbadge.vercel.app/api/shield/651319300166254592?style=flat-square&?compact=true&?logoColor=blue) (Alpha and Beta testing for me :neutral_face:)<br> -->
 
-* Last but not least, <b><a href="https://discord.com/users/291193188956241921">Scottywonderful</a></b> (Yes, myself, because I made this mostly from scratch and did my own ideas in it as well :cowboy_hat_face:)<br>
+* Last but not least, <b><a href="https://discord.com/users/291193188956241921">Scottywonderful</a></b> (Yes, myself, because I made this mostly from scratch and did my ideas in it as well :cowboy_hat_face:)<br>
 <!--![Scottywonderful](https://dcbadge.vercel.app/api/shield/291193188956241921?style=flat-square&?compact=true&?logoColor=purple)<br>-->
-Need any support then please join our discord <a href="https://discord.gg/GKbEFJZvz4">https://discord.gg/GKbEFJZvz4</a><br>
\ No newline at end of file
+Need any support then please join our discord <a href="https://discord.gg/GKbEFJZvz4">https://discord.gg/GKbEFJZvz4</a><br>
```

Comment:
No sure what change you did here as it shows there is a change but I don't see one.

Reply:
I guess just GitHub being GitHub.